rsconf/component/sirepo.py

```diff
@@ -271,16 +269,26 @@ def _install_dirs_and_files(self):
             )
 
     def _jupyterhublogin(self, z):
-        z.jupyterhub_enabled = "jupyterhublogin" in set(
-            self.j2_ctx.sirepo.feature_config.get("default_proprietary_sim_types", []),
-        ).union(set(self.j2_ctx.sirepo.feature_config.get("moderated_sim_types", [])))
+        z.jupyterhub_enabled = "jupyterhublogin" in self._in_sim_types(
```

Comment:
I think this is incorrect. jupyterhub_enabled could be true if jupyterhublogin is in sim_types. It does not have to be a proprietary or moderated sym type for this code to work. `_in_sim_types` should take a sym_type (jupyterhublogin) as an argument and return bool.

Reply:
3685a2ea3618ee935808a36d50a01190afebbc43